### PR TITLE
Pilot (Student Libraries): Allow levelbuilders to add libraries.

### PR DESCRIPTION
**Pull request by jmkulwik**
This is just the levelbuilder portion. The other portions are here: https://github.com/code-dot-org/code-dot-org/pull/28659 https://github.com/code-dot-org/code-dot-org/pull/28672 https://github.com/code-dot-org/code-dot-org/pull/28720

Spec for reference: https://docs.google.com/document/d/1jMLm_ghCshFen-h9vInAuXwEGXI-HPMjSP8dYeISfM0/edit?pli=1#

This code is largely experimental. If we go forward with Student Libraries, all code here will go through a second CR for production quality review. Review this PR to evaluate the following:

Isolation: Will the reviewed code affect parts of the product that are not part of the pilot?
Will the code fail in ways that will cause the pilot to be unsuccessful?
Is there an obviously easier or better way to implement the piloted feature?
Although the piloting process is still under review, I am following the procedures in this doc: https://docs.google.com/document/d/10QvRAVOTEenFJa1wwrY0Twkd0anTDpkkzNeGnT_HH0o/edit

### CODE REVIEW

File: dashboard/app/models/levels/blockly.rb [hunk 1]
```diff
@@ -45,6 +45,7 @@ class Blockly < Level
     ideal
     min_workspace_height
     step_speed
+    start_libraries
```
Comment (dmcavoy) [inline dashboard/app/models/levels/blockly.rb]: I thought blockly was the CSF levels. Should this go in applab.rb?

**Review by dmcavoy — COMMENTED** [review]
One question but I haven't been overly involved in this work

File: dashboard/app/models/levels/blockly.rb (hunk 1 above)
Comment (jmkulwik) [reply dashboard/app/models/levels/blockly.rb]: Thank you for catching that.

**Review by dmcavoy — APPROVED** [review]
LGTM :)

**Pull request merged**